ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/PackageInfoExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases.ui.revenuecatui.extensions
+
+import com.revenuecat.purchases.ui.revenuecatui.R
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.TemplateConfiguration
+import com.revenuecat.purchases.ui.revenuecatui.helpers.ApplicationContext
+import kotlin.math.roundToInt
+
+@SuppressWarnings("MagicNumber")
+internal fun TemplateConfiguration.PackageInfo.localizedDiscount(
```

Comment:
Can you add a test for this?
Also I wonder if this would be easier to find inside of `VariableDataProvider`

Reply:
Yes... Maybe would you add it as a extension there? The thing is that there's really no variable in this, that's why I didn't put it there. But I will move it as an extension for now

Reply:
> The thing is that there's really no variable in this,

This will be used for `{{ sub_relative_discount }}`.